counting_sort/counting_sort.py

```diff
@@ -0,0 +1,38 @@
+#Running time Complexity: O(n)
+#Auxillary Space: O(n)
+def count_occurences(a, c):
+    """
+    Input params: Array, intermediate variable
```

Comment:
{
"""
 Auxiliary Space: O(n)
    :param array: Iterable of elements
    :param left: left limit for merge sort
    :param right: right limit for merge sort
    :return: no returns, merges arrays.
    """
}
The above example is correct?